sdk/appcenter-crashes/src/main/java/com/microsoft/appcenter/crashes/Crashes.java

```diff
@@ -555,6 +560,33 @@ private void processPendingErrors() {
 
     }
 
+    private void processPendingBreakpadErrors() {
+        post(new Runnable() {
+
+            @Override
+            public void run() {
+                for (File logFile : ErrorLogHelper.getStoredBreakpadLogFiles()) {
+                    AppCenterLog.debug(LOG_TAG, "Process pending breakpad file: " + logFile);
+                    byte[] logfileContents = StorageHelper.InternalStorage.readBytes(logFile);
+
+                    /* Create our Breakpad dump attachment */
+                    ErrorAttachmentLog breakpadAttachment = ErrorAttachmentLog.attachmentWithBinary(logfileContents, "minidump.dmp", "application/octet-stream");
+                    List<ErrorAttachmentLog> list = new LinkedList<>();
+                    list.add(breakpadAttachment);
+
+                    /* Attach dump to NDK managed exception */
+                    ManagedErrorLog errorLog = ErrorLogHelper.createErrorLog(mContext, Thread.currentThread(), new NativeException(), Thread.getAllStackTraces(), mInitializeTimestamp, true);
+                    if(errorLog != null) {
+                        errorLog.getException().setWrapperSdkName("appcenter.ndk");
```

Comment:
i think there was a bad merge, i added a line here and it got removed. should also have

```
errorLog.getDevice().setWrapperSdkName("appcenter.ndk");
```

Reply:
@JessicaYeh I accidentally removed this when refactoring for the unit tests. Put this back in.